### PR TITLE
update Readme based on learning from #1985

### PR DESCRIPTION
**Pull request by graysky2**
Based on the time we spent trouble shooting in #1985, this can help others.  Also note that I removed the requirement for golangci-lint since the make process works just fine without it.  Perhaps it is needed for something else?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,8 +155,8 @@ Run `make init` to prepare the development environment.
 You will need this to build AdGuard Home:
 
  * [go](https://golang.org/dl/) v1.14 or later.
- * [node.js](https://nodejs.org/en/download/) v10 or later.
- * [golangci-lint](https://github.com/golangci/golangci-lint)
```
Comment (szolin) [inline README.md:159]: This is needed for Go devs (a code linting tool)

File: README.md [hunk 2]
```diff
@@ -155,8 +155,8 @@ Run `make init` to prepare the development environment.
 You will need this to build AdGuard Home:
 
  * [go](https://golang.org/dl/) v1.14 or later.
- * [node.js](https://nodejs.org/en/download/) v10 or later.
```
Comment (szolin) [inline README.md:158]: Well, I have node v10.16.2 and it works fine.

File: README.md [hunk 3]
```diff
@@ -165,7 +165,7 @@ Open Terminal and execute these commands:
 ```bash
 git clone https://github.com/AdguardTeam/AdGuardHome
 cd AdGuardHome
-make
+make -j1
```
Comment (szolin) [inline README.md:168]: We should try to fix the Makefile first.  I'll post an update here on this.

**Comment by graysky2** [comment]
I adjusted and rebased the PR based on your feedback above.  We can still wait on the `make -j1` part as you see fit.

File: README.md (hunk 3 above)
Comment (szolin) [reply README.md:171]: Resolved in 5188da60cfea2b1018e73dbf4d4d1b1342906352

**Comment by graysky2** [comment]
I verified that ( 5188da6) works on Arch's build system.  Rebased.

**Pull request closed without merging**